bento_wes/backends/wes_backend.py

```diff
@@ -252,6 +257,114 @@ def get_workflow_name_wdl(workflow_path: Path) -> str | None:
             # Invalid/non-workflow-specifying WDL file if false-y
             return workflow_id_match.group(1) if workflow_id_match else None
 
+    def _download_to_path(self, url: str, token: str, destination: str):
+        """
+        Download a file from a URL to a destination directory.
+        Bearer token auth works with Drop-Box and DRS.
+        """
+        with requests.get(
+            url,
+            headers={"Authorization": f"Bearer {token}"},
+            verify=self.validate_ssl,
+            stream=True
+        ) as response:
+            if response.status_code != 200:
+                raise RunExceptionWithFailState(
+                    STATE_EXECUTOR_ERROR,
+                    f"Download request to drop-box resulted in a non 200 status code: {response.status_code}"
+                )
+            with open(destination, "wb") as f:
+                # chunk_size=None to use the chunk size from the stream
+                for chunk in response.iter_content(chunk_size=None):
+                    f.write(chunk)
+        self.log_debug(f"Downloaded file at {url} to path {destination}")
+
+    def _download_input_files(self, inputs: str | list[str], token: str, run_dir: Path) -> str | list[str]:
+        if not inputs:
+            # Ignore empty imputs
+            return inputs
+
+        if isinstance(inputs, list):
+            return [self._download_input_file(f) for f in inputs]
```

Comment:
this causes an error because `_download_input_file` requires more arguments than are passed here.

Reply:
fixed